subresource-loading.md

```diff
@@ -237,20 +244,38 @@ When returning to the event loop, if there are any chunk IDs on the list to fetc
 - When the fetch returns from the server, map in all fetched chunk IDs and add them to the persistent cache.
 - Then, the original fetch can continue at the top of the algorithm. It should either be served by a mapped-in chunk or 404.
 
+#### Pros
+* Mixing in decelration of the resources in the bundle and scopes covered by it in ways that enable use of both techniques, without requiring hard-to-implement [Cache Digests](https://tools.ietf.org/html/draft-ietf-httpbis-cache-digest-05).
+* Decouples the resource number from the number of chunks in ways that enable higher scalability in cases of closely knit, very small modules.
+* Update flow doesn't require content-addressable URLs, which avoids problems with unnecessary cache invalidation of references, as well as "purge" like semantics when resource version changes.
+  - Unclear if that part would work properly in the "online URL enforcement" revalidation flow.
+
+#### Cons
+* Requires declaring full dependency map on each entry point, that includes all the chunks required by all the entry points. If developers will get that wrong, it can result in spurious downloads.
+* A "just download the entire bundle" fallback is not feasible, since the full bundle can be extremely large. Related, caches that ignores Vary will have have A Bad Time™. 
+* As the downloaded resources are likely to be significantly smaller than the the complete bundle, [compression benefits of subset recompression](https://dev.to/riknelix/fast-and-efficient-recompression-using-previous-compression-artifacts-47g5) are likely to be small. (See [results](https://dev.to/riknelix/fast-and-efficient-recompression-using-previous-compression-artifacts-47g5#results) for 90% removal)
```

Comment:
I think this phrasing could be difficult to understand for people who aren't deep in this space. I'd say something more direct, like, "It could be prohibitively slow to dynamically compress collections of chunks to a high quality, as there are certain techniques that work quickly in the case of a large subset, but not as well in a smaller subset (see [results])".

Reply:
Sure!